cmd/collector/app/sanitizer_v2/empty_service_name_sanitizer.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2024 The Jaeger Authors.
```

Comment:
`cmd/collector/app/sanitizer_v2/` location doesn't make sense. We can put these into `cmd/jaeger/internal/sanitizer`

Reply:
Yes, thanks for reminding. Before I had thought to add it at  `cmd/jaeger/internal/sanitizer`. But I am not sure it could be a appropriate place and just added here below the existing one.

Reply:
Yes let's go with this `cmd/jaeger/internal/exporters/storageexporter/sanitizer` as we are going to integrate to exporter, it seems good place to keep it